dependencies.props

```diff
@@ -1,10 +1,39 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+    <!-- For detailed explanation of the reference versions see:
+
+            https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+            https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+        These ref versions are pulled from https://github.com/dotnet/versions.
+    -->
+    <PropertyGroup>
+        <BuildToolsCurrentRef>bd37cedea957c312fb7d3d681f416d51227d4df1</BuildToolsCurrentRef>
```

Comment:
[At this commit](https://github.com/dotnet/versions/blob/bd37cedea957c312fb7d3d681f416d51227d4df1/build-info/dotnet/buildtools/master/Last_Build_Packages.txt), the feed package is at `1.0.0-prerelease-02124-02`. It'll auto-upgrade fine, but right now they're out of sync. I suggest just grabbing the latest from dotnet/versions master (`c6906dcb2309085e414c5b9691de84fbe2bf5c86`) then running `/t:UpdateDependencies` to get it in sync (and to latest).